.github/workflows/go.yaml

```diff
@@ -0,0 +1,42 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Setup Go
+      uses: actions/setup-go@v2-beta
+      with:
+        go-version: '^1.14'
+        
+    - name: Install goimports
+      run: go get golang.org/x/tools/cmd/goimports
+
+    - name: Check code
+      uses: actions/checkout@v2
+    
+    - name: goimports
+      run: test -z "$($(go env GOPATH)/bin/goimports -l .)"
+        
+    - name: Fix goimports on fail
+      run: echo Run goimports -w .
+      if: failure()
+
+    - name: License check
```

Comment:
Looks like this is failing for a bunch of files that I think are OK?

Reply:
This seems tricky to get right. We should be able to reuse an existing check. So, let's just remove the license check for now.